<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopologyMapWindow</class>
 <widget class="QMainWindow" name="TopologyMapWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuPerspective">
     <property name="title">
      <string>Perspective</string>
     </property>
     <addaction name="actionTop"/>
     <addaction name="actionSide"/>
     <addaction name="actionFront"/>
    </widget>
    <addaction name="menuPerspective"/>
    <addaction name="actionCircle"/>
    <addaction name="actionForce_Directed"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuview"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="text">
    <string>Top</string>
   </property>
  </action>
  <action name="actionSide">
   <property name="text">
    <string>Side</string>
   </property>
  </action>
  <action name="actionFront">
   <property name="text">
    <string>Front</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionForce_Directed">
   <property name="text">
    <string>Force-Directed</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
